runtime/base/strings/string_number_conversions.cc

```diff
@@ -0,0 +1,238 @@
+// Copyright (c) 2012 The Chromium Authors. All rights reserved.
+// Copyright (c) 2015 Intel Corporation. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "xwalk/runtime/base/strings/string_number_conversions.h"
```

Comment:
Why copying the code in `src/base/strings/string_number_conversions.cc` instead of re-use it?